packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -204,7 +201,11 @@ const Button = ({
           fullWidth={isFullWidth}
           id={id}
           onClick={onClick}
-          ref={localButtonRef}
+          ref={(element) => {
+            localButtonRef.current = element;
+            //@ts-expect-error ref is not an optional prop on the props context type
+            muiProps?.ref?.(element);
+          }}
```

Comment:
Can we `useCallback` this ref?

Reply:
Which one?

Reply:
This whole function:
```tsx
(element) => {
  localButtonRef.current = element;
  //@ts-expect-error ref is not an optional prop on the props context type
  muiProps?.ref?.(element);
}
```

Reply:
@KevinGhadyani-Okta Maybe, what would it's dependencies be though? This is all already inside a useCallback render function